examples/worlds/spherical_coordinates.sdf

```diff
@@ -60,21 +75,237 @@ ign service -s /world/spherical_coordinates/set_spherical_coordinates \
       name="ignition::gazebo::systems::SceneBroadcaster">
     </plugin>
 
-    <!-- Processes pose commands -->
-    <plugin
-      filename="ignition-gazebo-physics-system"
-      name="ignition::gazebo::systems::Physics">
-    </plugin>
-
     <!-- Set the coordinates for the world origin -->
     <spherical_coordinates>
       <surface_model>EARTH_WGS84</surface_model>
       <world_frame_orientation>ENU</world_frame_orientation>
-      <latitude_deg>-22.9</latitude_deg>
-      <longitude_deg>-43.2</longitude_deg>
+      <latitude_deg>-22.986687</latitude_deg>
+      <longitude_deg>-43.202501</longitude_deg>
       <elevation>0</elevation>
       <heading_deg>0</heading_deg>
     </spherical_coordinates>
 
+    <gui fullscreen="0">
+
+      <!-- 3D scene -->
+      <plugin filename="MinimalScene" name="3D View">
+        <ignition-gui>
+          <title>3D View</title>
+          <property type="bool" key="showTitleBar">false</property>
+          <property type="string" key="state">docked</property>
+        </ignition-gui>
+
+        <engine>ogre2</engine>
+        <scene>scene</scene>
+        <ambient_light>0.4 0.4 0.4</ambient_light>
+        <background_color>0.8 0.8 0.8</background_color>
+        <camera_pose>-6 0 6 0 0.5 0</camera_pose>
```

Comment:
Can we set the camera pose so that when the box is moved in the scene, it matches the direction of the NavSatMap display for a better OOBE :sparkles: ?

```suggestion
        <camera_pose>0 -6 8 0 0.8 1.56</camera_pose>
```

Reply:
Ah good call, I had done it for the gif but not for the example. Done in 4644e7e